src/stream/src/executor/monitor/streaming_stats.rs

```diff
@@ -297,6 +305,14 @@ impl StreamingMetrics {
             )
             .unwrap();
 
+        let dispatcher_count = register_guarded_int_gauge_vec_with_registry!(
+            "stream_dispatcher_count",
+            "Total number of dispatchers",
+            &["fragment_id", "downstream_fragment_id"],
+            registry
+        )
+        .unwrap();
```

Comment:
I think this is exactly equal to the parallelism (i.e. number of actors) of this fragment, no need to introduce a new metric.

Reply:
You're right as we currently don't support multiple-edges between two fragments. But is there existing metrics for "parallelism"? 👀

Reply:
> But is there existing metrics for "parallelism"? 👀

We can retrieve this from metrics `actor_info` of which labels are `"actor_id", "fragment_id", "compute_node"`, but is this also an actor-level metrics with high cardinality? 🥵🥵

Reply:
Indeed. Let's introduce a new metrics for "parallelism (i.e. number of actors) of this fragment"